app/scripts/directives/notifications/notificationDrawerWrapper.js

```diff
@@ -172,7 +173,7 @@
       };
 
       var notificationWatchCallback = function(event, notification) {
-        if(!notification.showInDrawer) {
+        if(!notification.showInDrawer || EventsService.isCleared(notification.id)) {
```

Comment:
Shouldn't we be using `notification.uid` for cleared? We seem to use that in other places.

Reply:
The notification in this callback doesn't have the `uid`; the `uid` is set if it gets past this check.

Reply:
Yeah, that relates to my comment above.  Should always be consistent in the drawer.  Think I settled on `uid` originally because of `apiEvents` first.

Reply:
But I don't think we're not checking the right ID.... If we're clearing `uid` but checking `id`, isn't this check just wrong?

Reply:
Ok, addressed this logic issue.